### PR TITLE
Copyedit the new Ruby Arrays lesson

### PR DESCRIPTION
**Pull request by leila-alderman**
My edits for the new Arrays lesson.

### CODE REVIEW

File: new_course/1_programming_fundementals/arrays.md [hunk 1]
```diff
@@ -35,16 +37,19 @@ Array.new(3, Array.new) #=> [[], [], []]
...
 
+str_array[0]            #=> "This"
+str_array[1]            #=> "is"
+str_array[4]            #=> "array"
 str_array[-1]           #=> "array"
 str_array[-2]           #=> "small"
 ~~~
 
-Finally, Ruby provides the `first` and `last` methods, which should be self-explanatory. What may not be obvious, however, is that these methods can take an argument that will return a *new* array.
+Finally, Ruby provides the `first` and `last` array methods, which should be self-explanatory. What may not be obvious, however, is that these methods can take an integer argument (`myArray.first(n)` or `myArray.last(n)`), which will return a new array that contains the first or last `n` elements of `myArray`.
```
[the remark this answers is not in the document]
Comment (leila-alderman) [reply new_course/1_programming_fundementals/arrays.md]: My thought with having both was to be able to add a "respectively" at the end of the sentence, which I apparently forgot to do. I realize it's probably pretty obvious, but I like the added clarity.

File: new_course/1_programming_fundementals/arrays.md [hunk 2]
```diff
@@ -54,31 +59,42 @@ str_array.first(2)      #=> ["This", "is"]
...
-
 num_array.pop             #=> 5
+num_array                 #=> [1, 2, 3, 4]
 ~~~
 
-The methods `shift` and `unshift` can also be used to operate at the beginning of an array. While `shift` will remove the first element of an array and return that element (much like `pop`), `unshift` will add elements to the beginning of an array and return that array.
+The methods `shift` and `unshift` are used to add and remove elements at the beginning of an array. The `unshift` method adds elements to the beginning of an array and returns that array (much like `push`). The `shift` method removes the first element of an array and returns that element (much like `pop`).
 
 ~~~ruby
 [1, 2, 3, 4].unshift(0)   #=> [0, 1, 2, 3, 4]
```
[the remark this answers is not in the document]
Comment (leila-alderman) [reply new_course/1_programming_fundementals/arrays.md]: Agreed!

Is it worth having an example with this format in this lesson to show that it's possible to do it this way?

**Comment by leila-alderman** [comment]
I don't have a strong opinion on `method` vs. `#method`. @KevinMulhern?

**Comment by KevinMulhern** [comment]
`#method` is used a lot in ruby documentation, I think we should stick with that convention to get students comfortable with it.

File: new_course/1_programming_fundementals/arrays.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (leosoaivan) [reply new_course/1_programming_fundementals/arrays.md]: I guess we could. I'm honestly 50/50 on this. @KevinMulhern?

[the remark this answers is not in the document]
Comment (leila-alderman) [reply new_course/1_programming_fundementals/arrays.md]: I just changed this example to the other format. There are some examples later in this lesson that use the `[1, 2, 3].join` format.

**Pull request merged**